code/modules/admin/admin_verbs.dm

```diff
@@ -1178,14 +1178,14 @@ var/list/fun_images = list()
 	else
 		delay = inputDelay
 
-	var/confirm2 = alert("Make everyone invisible? (Literally every mob)", "Invisible Mobs?", "No", "Yes")
+	var/confirm2 = tgui_alert(usr, "Make everyone invisible? (Literally every mob)", "Invisible Mobs?", list("No", "Yes"))
```

Comment:
Can you swap the order of all the buttons in this command? Standard is cancel on the right, but all these are `Yes` on the right.

Reply:
Sure, you want me to do it for the others as well? There's about 4 others that also are doing this.

Reply:
Yeah, all of them would be good.